### PR TITLE
fix: review updates

### PR DESCRIPTION
**Pull request by nhardman**

### CODE REVIEW

File: en/starter/generator/command_line_tools_for_linux.md [hunk 1]
```diff
@@ -92,11 +92,11 @@ The generator prompts you for the following information:
...
 
--   Select services (for scaffold a starter application) to generate boilerplate for; these will be Bluemix services if you selected the Bluemix cloud deployment capability, otherwise they will be local services. The available Bluemix services are Cloudant®, Redis, Object Storage, AppID and Auto-scaling; the local services are CouchDB and Redis. The tool prompts you for additional settings and automatically installs the appropriate Swift packages using the Swift Package Manager.
+-   Select services (for scaffold a starter application) to generate boilerplate for; The available IBM Cloud services are Cloudant®, Redis, Object Storage, AppID and Auto-scaling; CouchDB and Redis are also available as local services. The tool prompts you for additional settings and automatically installs the appropriate Swift packages using the Swift Package Manager.
```
Comment (tunniclm) [inline en/starter/generator/command_line_tools_for_linux.md]: "local services" do not exists any more.

File: en/starter/generator/core_concepts.md [hunk 2]
```diff
@@ -239,10 +239,6 @@ This service uses the [SwiftMetrics package](https://github.com/RuntimeTools/Swi
 
 The connection details for this client are loaded by the configuration boilerplate code and are passed to the SwiftMetrics auto-scaling client.
 
-### Non-IBM Cloud services
-
-Non-IBM Cloud services can be hosted anywhere including your local machine.
-
```
Comment (tunniclm) [inline en/starter/generator/core_concepts.md]: You didn't remove the whole section -- local CouchDB and Redis are not supported any more, they need to be removed also.

File: en/starter/generator/command_line_tools_for_linux.md (hunk 1 above)
Comment (nhardman) [reply en/starter/generator/command_line_tools_for_linux.md]: done

File: en/starter/generator/core_concepts.md (hunk 2 above)
Comment (nhardman) [reply en/starter/generator/core_concepts.md]: done

**Pull request merged**